README.md

```diff
@@ -22,29 +22,28 @@ repositories {
 }
 
 dependencies {
-  compile 'com.github.paolorotolo:appintro:4.0.0'
+    compile 'com.github.paolorotolo:appintro:4.0.0'
 }
 ```
 
 Create a new **Activity that extends AppIntro**:
 
 ```java
-public class MyIntro extends AppIntro {
-
+public class IntroActivity extends AppIntro {
     @Override
-    protected void onCreate(Bundle savedInstanceState) {
+    protected void onCreate(@Nullable Bundle savedInstanceState) {

```

Comment:
It looks like `AppIntroBase` already has this annotation: https://github.com/PaoloRotolo/AppIntro/blob/master/library/src/main/java/com/github/paolorotolo/appintro/AppIntroBase.java#L75